frontend/src/entry/model/releases/EditableRelease.tsx

```diff
@@ -219,25 +219,27 @@ export default function EditableRelease({ release, isEdit, onIsEditChange }: Edi
 
   return (
     <Stack spacing={2}>
-      <EditableFormHeading
-        heading={
-          <div>
-            <Typography fontWeight='bold'>Release name</Typography>
-            <Typography>{`${model.name} - ${release.semver}`}</Typography>
-          </div>
-        }
-        editButtonText='Edit Release'
-        deleteButtonText='Delete Release'
-        showDeleteButton
-        isEdit={isEdit}
-        isLoading={isLoading}
-        onEdit={handleEdit}
-        onCancel={handleCancel}
-        onSubmit={handleSubmit}
-        onDelete={() => setOpen(true)}
-        errorMessage={errorMessage}
-        isRegistryError={isRegistryError}
-      />
+      {(!model.settings.mirror || model.settings.mirror?.sourceModelId) && (
```

Comment:
We'll still want the heading so the release name is shown. Are you doing this to hide the edit/delete buttons? If so, we should instead conditionally render those buttons. You'll just need to pass through a `readOnly` prop and handle it within `EditableFormHeading`.